program/c/src/oracle/oracle.h

```diff
@@ -394,7 +413,27 @@ typedef struct cmd_upd_test
   uint64_t     conf_[PC_COMP_SIZE];
 } cmd_upd_test_t;
 
+#ifdef PC_PYTHNET
+
+// Replace Solana component size's contribution with Pythnet's.
+//
+// Note: Make sure to account here for all PC_COMP_SIZE-sized arrays in the struct.
+#define PC_EXPECTED_CMD_UPD_TEST_T_SIZE_PYTHNET (560 \
```

Comment:
Since this instruction is no longer used, I'd just get rid of the test altogether